content/posts/macinplanck-configuration/index.md

```diff
@@ -0,0 +1,326 @@
+
+Also, any action that I explicitly didn't set in a given layer, I marked it as "None". This was done because, by default, the Oryx configurator marks other keys in the layer "transparent" to let the key fall through to the `Base` layer. I do this in all layers except the `Base` layer (that layer is full anyway, so it doesn't matter). This was done because I didn't want to encounter any behavior that I didn't explicitly program. The `TT()` keys are not set to "None" because they still have a function; pressing one of those keys again will take you into the `Adjust` layer or will take you back to the `Base` layer, depending on which layer you're in.
+
+The last "big thing" to note here (because it's common in other layers too) is that at this point, I've designated the ESC key as my "escape hatch" in all layers that could be considered "not transient" (meaning that I can press a key and that layer could remain active ever if I release the key). This key is mapped to `TO(1)` everywhere that is relevant, which brings you back to the `Base` layer. There are other keys in other layers that do this, but the ESC key is supposed to be my "safe key" that gets me out of anywhere I might lose myself in. Pressing ESC many times is also fine in pretty much all situations, since even if I get into the `Base` layer, it's just mapped to _actual_ ESC, which doesn't really ever do anything destructive.
+
+Other things to note here:
+
+- I removed all function keys, since I basically never use them.
+- I removed various other keys like `PgUp`/`PgDown` because I don't really find them to be all that useful (I have other shortcuts that accomplish the same thing) and I don't think that I'd use the Non-US characters.
+- I brought back `Enter` and `Backspace` because they "made sense" to keep (I was gaining nothing by having them removed like it is in the default configuration).
+- I persisted the same modifier keys from the `Base` layer into both layers because I have a lot of application-specific shortcuts that rely on using these modifiers in conjunction with characters/symbols that are in these layers.
+
+The rest of the layout should be fairly obvious, so I'm not going to enumerate all of the mapping. 3 layers down, 9 to go.
+
+### The `Adjust` Layer
+
+![adjust layer](./3-adjust.png)
+
+I have the least to say about this layer. Note that we have the same "escape hatch", but all other keys remain the same as the default configuration. If you have any ideas what else I should put here, please let me know!
```

Comment:
> If you have any ideas what else I should put here, please let me know!

"If you have any ideas as to what else", perhaps? As written it feels a little weird to say? That could be just me though.

Reply:
Nah makes sense, adding!